src/Server/Player/OrangeLogic.py

```diff
@@ -29,11 +28,14 @@ def _process_requests(self, network_messages):
                     Server.get_instance().send_all(mes)
 
     def _attack(self):
-        if super()._attack() and not self._is_flying:
+        if self._can_attack and not self._is_flying:
+            super()._attack()
             self._is_attacking = True
             self._add_ground_directed_force(self._attack_strength, self.my_dir())
             self._impact()
             return True
+        else:
+            print("you cant attack")
         return False
```

Comment:
Again, return values misleading. (only that this is my implementation :D )

Reply:
Print message doesn't seem to be informative - who can't attack? Apple/Orange/PlayerID
If it was just for testing purposes, then should be removed.